packages/components/src/popover/README.md

```diff
@@ -123,6 +123,20 @@ Opt-in prop to show popovers fullscreen on mobile, pass `false` in this prop to
  - Required: No
  - Default: `false`
 
+### getAnchorRect
```

Comment:
Are we certain about the future of `getAnchorRect` before documenting it? Perhaps we could work towards only supporting `anchorRect`.

Reply:
> Are we certain about the future of `getAnchorRect` before documenting it? Perhaps we could work towards only supporting `anchorRect`.

I don't mind to remove the documentation, if even just to keep the changes well-contained. I don't have much a problem with this prop, at least in that it's a useful way to also base the computed anchor rect relative the passed default `anchor` argument. I'm also not entirely comfortable with the prospect of removing functionality based on the absence of it from documentation either.